Backend/dev/test-tool/local-api/server.py

```
@@ -933,6 +939,84 @@ def _is_localhost(host: str) -> bool:
     return (host or "").strip() in ("localhost", "127.0.0.1", "::1", "")
 
 
+# ── Registry helpers ──
+
+
+def _remote_read_registry(ssh_user: str, host: str, port: int, identity: str | None) -> dict:
+    """Read the devbox registry JSON from the remote host via SSH."""
+    cmd = f'cat {REGISTRY_FILE} 2>/dev/null || echo \'{{"users": {{}}}}\''
+    argv = _ssh_argv(ssh_user, host, port, identity, want_tty=False) + [cmd]
```

Ruff (0.15.13)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)